frontend/app/routes/$lang+/_public+/apply+/$id+/communication-preference.tsx

```diff
@@ -79,49 +89,114 @@ export async function action({ request, params }: ActionFunctionArgs) {
 
   const formSchema = z
     .object({
-      preferredLanguage: z.string({ required_error: 'empty-language' }),
-      preferredMethod: z.string({ required_error: 'empty-method' }),
-      email: z.string().min(1, { message: 'empty-email' }).email({ message: 'invalid-email' }).optional(),
-      confirmEmail: z.string().min(1, { message: 'empty-confirm-email' }).optional(),
-      emailForFuture: z.preprocess((arg) => (arg === '' ? undefined : arg), z.string().email({ message: 'invalid-email' }).optional()),
-      confirmEmailForFuture: z.string().optional(),
+      preferredLanguage: z.string().min(1, { message: t('apply:communication-preference.error-message.empty-language') }),
+      preferredMethod: z.string().min(1, { message: t('apply:communication-preference.error-message.empty-method') }),
```

Comment:
Should these validators check the content of the string? What if someone sends `es` for his preferred language?

(or `'      '`)